src/SessionInterface.php

```diff
@@ -68,4 +68,9 @@ public function regenerate(): self;
      * true if the instance was produced via regenerate().
      */
     public function isRegenerated(): bool;
+
+    /**
+     * Retrieve the session identifier.
+     */
+    public function getId(): string;
```

Comment:
It'd be nice if we could improve type precision by introducing `@return non-empty-string`